Core/Main/elxTransformixFilter.h

```diff
@@ -153,10 +153,7 @@ class ELASTIXLIB_API TransformixFilter : public itk::ImageSource< TMovingImage >
   void operator=( const Self & );    // purposely not implemented
 
   /** IsEmpty. */
-  virtual bool IsEmpty( const InputImagePointer inputImage );
-
-  /** Let transformix handle input verification internally. */
-  virtual void VerifyInputInformation( void ) ITK_OVERRIDE {};
+  bool IsEmpty( const InputImagePointer inputImage );
```

Comment:
Changing the declaration of `IsEmpty` seems unrelated to the rest of the commit, but if you want to change it, maybe you could even declare it `static`!

Reply:
Static is a good idea.